examples/README.md

```diff
@@ -9,25 +9,25 @@ This page contains a list of example codes written with Optuna.
 
 ### Examples with ML Libraries
 
-* [Scikit-learn](./sklearn_simple.py)
+* [Scikit-learn](./sklearn/sklearn_simple.py)
```

Comment:
The `OptunaSearchCV` used in `examples/optuna_search_cv_simple.py` is the integration module with `scikit-learn`. How about placing it into `examples/sklearn` and adding here?

Reply:
The name of class is `OptunaSearchCV`, so how about naming the `sklearn_optuna_search_cv_simple.py`? In addition, how about adding the file to this line.